### PR TITLE
Manually update version: v1.3

### PR DESCRIPTION
**Pull request by adrn**
Checklist
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by adrn** [comment]
Because I renamed the package, the auto PR bot didn't re-open a PR to tick the version up, so this is my attempt to do it manually! @mwcraig - can you take a look and let me know if you have any ideas?

**Comment by adrn** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,11 +8,11 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
```
Comment (mwcraig) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,11 +8,11 @@ package:
...
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 24bc8333046ae05bd4dbb00512525033df128f07f33876d43e5ecbcc4b7d06e1
+  sha256: 38d3c4e4ad39fc1b061fd23832df0552a2640be2e468f2d3799349e0e0374d75
 
 build:
   number: 0
-  skip: true  # [win or py<36]
+  skip: true  # [win or py<37]
   script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```
Comment (mwcraig) [inline recipe/meta.yaml]:
```suggestion
  script: {{ PYTHON }} -m pip install --no-deps -vv .
```

**Review by mwcraig — CHANGES_REQUESTED** [review]
Just a few minor changes, none of which are critical. As long as the recipe is being updated we might as well modernize a couple things.

**Pull request merged**